docs/DeveloperGuide.md

```diff
@@ -674,86 +760,74 @@ testers are expected to do more *exploratory* testing.
 
 ### Launch and shutdown
 
-1. Initial launch
-
+1. Initial launch:
    1. Download the jar file and copy into an empty folder
-
    1. Double-click the jar file 
-   
+    
    Expected: Shows the GUI with a set of sample students and classes. The window size may not be optimum on certain resolutions.
-
-1. Saving window preferences
-
+ 
+1. Saving window preferences:
    1. Resize the window to an optimum size. Move the window to a different location. Close the window.
-
    1. Re-launch the app by double-clicking the jar file.
    
    Expected: The most recent window size and location is retained.
 
 ### Deleting a student
 
-1. Deleting a student while all students are being shown
-
+1. Deleting a student while all students are being shown:
    1. Prerequisites: List all students using the `list` command. Multiple students in the list.
-
    1. Test case: `delete 1`<br>
       Expected: First contact is deleted from the list. Details of the deleted contact shown in the status message. Timestamp in the status bar is updated.
-
    1. Test case: `delete 0`<br>
       Expected: No student is deleted. Error details shown in the status message. Status bar remains the same.
-
    1. Other incorrect delete commands to try: `delete`, `delete x`, `...` (where x is larger than the list size)<br>
       Expected: Similar to previous.
 
 ### Adding sessions
 
-1. Adding a session to a class
-
+1. Adding a session to a class:
    1. Prerequisites: The class `CS1231S` exists in TA-Assist without any sessions assigned. 
       TA-Assist is currently in focus mode and is focusing on the `CS1231S` class.
-
    1. Test case: `adds s/Tut 1`<br>
       Expected: A session named `Tut 1` is added with the current system date and is displayed on the session list.
-
    1. Test case: `adds s/Tut 1 d/2022-01-01`<br>
       Expected: A session named `Tut 1` is added with its date set to `1st January 2022` and is displayed on the session list.
-
    1. Test case: `adds s/Lab 1 d/2019-02-29`<br>
       Expected: Session is not created as `29th February 2019` is not a valid date.
+   <p></p> <!--- Empty paragraph to add spacing between sections -->
 
-1. Batch adding sessions to a class
-
+1. Batch adding sessions to a class:
    1. Prerequisites: The class `CS1231S` exists in TA-Assist without any sessions assigned.
    TA-Assist is currently in focus mode and is focusing on the `CS1231S` class.
-
    1. Test case: `adds s/Tut 2 s/Lab 2`<br>
       Expected: Two sessions named `Tut 2` and `Lab 2` is added with the current system date and is displayed on the session list.
-
    1. Test case: `adds s/Tut 3 s/Lab 3 d/2000-01-01`<br>
       Expected: Two sessions named `Tut 3` and `Lab3` is added with its date set to `1st January 2000` and is displayed on the session list.
```

Comment:
```suggestion
      Expected: Two sessions named `Tut 3` and `Lab 3` is added with its date set to `1st January 2000` and is displayed on the session list.
```

Reply:
Oops, I just realized this was probably from my PR, thanks for the spot!